airbyte-cdk/java/airbyte-cdk/datastore-bigquery/src/main/kotlin/io/airbyte/cdk/db/bigquery/TempBigQueryJoolDatabaseImpl.kt

```diff
@@ -14,21 +14,20 @@ import org.jooq.exception.DataAccessException
 import org.jooq.impl.DefaultDSLContext
 
 /** This class is a temporary and will be removed as part of the issue @TODO #4547 */
-class TempBigQueryJoolDatabaseImpl(projectId: String?, jsonCreds: String?) : Database(null) {
-    val realDatabase: BigQueryDatabase
-
-    init {
-        realDatabase = createBigQueryDatabase(projectId, jsonCreds)
-    }
+class TempBigQueryJoolDatabaseImpl(
+    projectId: String?,
```

Comment:
doublechecking: these are intentionally still nullable?

Reply:
yeah, this was about removing the trivial ones. Removing all of them is going to be a whole lot harder...